context.json

```diff
@@ -21,15 +21,15 @@
           "@id": "prov:endedAtTime",
           "@type": "xsd:dateTime"
         },
-        "wasGeneratedBy" : {
+        "generatedBy" : {
           "@id": "prov:wasGeneratedBy",
           "@type": "@id"
         },
         "wasAttributedTo" : {
```

Comment:
Could we use the same approach for all prov terms, e.g. here `wasAttributedTo` --> `attributedTo` ?

Reply:
Sure, I just made the same changes for `attributedTo` and `informedBy`. However, I wonder what is their interest? I can't find them in the bids-prov specification.